io.github.brunofin.NotionDesktop.yml

```diff
@@ -0,0 +1,67 @@
+app-id: io.github.brunofin.NotionDesktop
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+
+base: org.electronjs.Electron2.BaseApp
+base-version: "22.08"
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node14
+
+build-options:
+  append-path: /usr/lib/sdk/node14/bin
+
+finish-args:
+  # D-Bus
+  - --share=ipc
+  # Xorg and Pulseaudio
+  - --socket=x11
+  - --socket=pulseaudio
+  # Network access
+  - --share=network
+  # Camera and OpenGL rendering
```

Comment:
why do yo need camera?

Reply:
you don't need openGL anymore?

(give it's electron you likely do)

Reply:
@hfiguiere I don't need the camera access in Notion. About OpenGL I am not sure, I build locally without `--device=all` and it seems to work the same. If you think it would be best to put that back, please let me know.

Reply:
If you don't need it seems fine